lumps/dehacked/dehacked.txt

```diff
@@ -639,6 +639,22 @@ OB_MPBFG_SPLASH=%o couldn't hide from %k's SKAG.
 OB_MPTELEFRAG=%o stood where %k was teleporting.
 OB_RAILGUN=%o was railed by %k.
 OB_MPBFG_MBF=%o had %p ass kicked by %k's SKAG!
+# Quit messages.
+QUITMSG = AGM's scientists are planning an even worse nightmare.\nPress Y if you don't care.
+QUITMSG1 = Why would you...
+QUITMSG2 = Hey buddy, there's still much work to be done.\nAre you truly sure you want to quit?
+QUITMSG3 = Press N to keep annihilating aliens.\nPress Y to surrender.
```

Comment:
`annihilating` is now repeated twice. How about using `decimating` here?

Reply:
Reduce by 1/10?